src/components/InviteFriends.js

```diff
@@ -21,14 +32,33 @@ const InviteFriends = (props) => {
     textFieldRef.current.select()
   }
 
+  const classes = useStyles()
   return (
-    <TextField
-      id="refer-friend-input"
-      inputRef={textFieldRef}
-      onClick={highlightReferralUrl}
-      value={referralUrl}
-      label={label}
-    />
+    <div className={classes.root}>
+      <TextField
+        id="refer-friend-input"
+        inputRef={textFieldRef}
+        onClick={highlightReferralUrl}
+        value={referralUrl}
+        label={label}
+      />
+      <SocialShare
+        url={referralUrl}
+        iconSize={24}
+        FacebookShareButtonProps={{}}
+        RedditShareButtonProps={{
+          title: '//TODO: fill in copy here',
+        }}
+        TumblrShareButtonProps={{
+          title: '//TODO: fill in copy here',
+          caption: '//TODO: fill in copy here',
+        }}
+        TwitterShareButtonProps={{
+          title: '//TODO: fill in copy here',
+          related: ['@TabForACause'],
```

Comment:
TODO: @amaliwanag2 update (or remove) these props before merging.

Reply:
@jedtan Feel free to do this. Remove all props except `url` and `iconSize`, then go ahead and merge.